35 wireframe to code

vibqetowi opened the pull request:
Reopening edwards PR

kokkuri3 committed:
CampusMap and Navbar styling

kokkuri3 committed:
Upload Image Asset

vibqetowi committed:
Update CODEOWNERS (#130)

vibqetowi committed:
Create dependabot.yml (#133)

vibqetowi committed:
Create sonarcloud.yml (#134)

vibqetowi committed:
Vibqetowi patch 1 (#135)

* Create sonarqube.yml

* merge

* updated readme

vibqetowi committed:
Vibqetowi patch 1 (#136)

* Create sonarqube.yml

* merge

* updated readme

vibqetowi committed:
add meeting notes 2025-02-12 (#147)

vibqetowi committed:
Meeting notes 1 (#148)

* add meeting notes 2025-02-12

* push release report

vibqetowi committed:
Documentation (#149)

* sync to pull

* added doc

dependabot[bot] committed:
Bump @babel/core from 7.26.7 to 7.26.8 in /src (#154)

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.26.7 to 7.26.8.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.26.8/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

emvillama committed:
refactoring _layout.tsx to utilize globalStyles.ts instead of inline

emvillama committed:
refactoring CampusMap.tsx to utilize globalStyles.ts instead of inline

emvillama committed:
refactoring ClassSchedule.tsx to utilize globalStyles.ts instead of inline

emvillama committed:
refactoring FindBuilding.tsx to utilize globalStyles.ts instead of inline

emvillama committed:
refactoring Home.tsx to utilize globalStyles.ts instead of inline (Home tab no longer in use can be deleted)

emvillama committed:
added import/export for Concordia_Small_Logo

emvillama committed:
created globalStyles.ts and stored styles

emvillama committed:
changed initial route to CampusMap and fixed menu not showing up in tabs

kokkuri3 committed:
Merge branch 'main' into 35-wireframe-to-code

vibqetowi committed:
added MVVM refactorings to fix compilation errors

vibqetowi committed:
Merge branch 'dev' of https://github.com/AsifAliKhan2001/Minicap into 35-wireframe-to-code

vibqetowi committed:
refactor colors

HudsonLu reviewed (dismissed):
Nice refactoring of CSS Inline + UI design added. Before merging, merge the branch with dev to avoid any conflicts. Good job! Something to work on would be accessibility. For some reasons, using Pixel 7 API 35, it shows
![image](https://github.com/user-attachments/assets/b6563a37-0356-45b8-96eb-3356a7b826d0)
while in another device emulator, it shows correctly:
![image](https://github.com/user-attachments/assets/a5e1875f-0559-4764-81a6-db1f27a58d4a)
Otherwise, it looks good.

allaye4 reviewed (dismissed):
I like the look of the UI. As @HudsonLu mentioned we'll have to make sure the layout is responsive. Aside from that it looks nice.

vibqetowi reviewed (commented):
as a bit of extra reusability, I propose creating a set of defined "brand color" variables and reusing them as

background-color: var(--concordia-red) for instance

see another repo of mine for reference https://github.com/vibqetowi/mway-task-2-demo/blob/main/assets/css/style.css

emvillama committed:
added autoclose hamburger
changed hamburger active color
added relative measures use
re-added my location button
added color variables

allaye4 commented:
> as a bit of extra reusability, I propose creating a set of defined "brand color" variables and reusing them as
> 
> 
> 
> background-color: var(--concordia-red) for instance
> 
> 
> 
> see another repo of mine for reference https://github.com/vibqetowi/mway-task-2-demo/blob/main/assets/css/style.css

I like this idea. It'll make it much easier for the upcoming UI issues.

vibqetowi reviewed (commented):
good work on color definition

allaye4 commented:
@emvillama there are some merge conflicts to be resolved

emvillama committed:
Merge branch 'dev' into 35-wireframe-to-code
